app.js

```diff
@@ -1,10 +1,12 @@
 const express = require('express');
 const { handlePsqlErrors, handleCustomErrors, handleServerErrors } = require('../be-nc-news/errors/error-handling')
-const { getApi } = require('./controllers/api.controller');
+const { getApi } = require('./controllers/api.controller')
 const { getAllTopics } = require('./controllers/topics.controller');
 
 const app = express();
 
+app.use(express.json());
```

Comment:
Please remove this line of code. We should only have code that is relevant to the ticket we're working on. For this ticket express.json() is not required 👀